### PR TITLE
[WIP] ui: update ja_JP locale

### PR DESCRIPTION
**Pull request by tajisoft**
### Description

Hello dev team. I am a Japanese user. I would like to update the Japanese translation. I have one question to ask. The translated text in the json file is a Unicode string. Is it okay to decode this into a Japanese string? That makes translation conversion easier. I'm sorry I couldn't find a channel to discuss and asked a question on PR.
This is still WIP, so I'm using Draft PR.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
WIP

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan ui

**Comment by rohityadavcloud** [comment]
Hi @tajisoft yes I think using unicode string should be completely okay - we can as part of your PR change check the QA url for the same.

**Review by DaanHoogland — COMMENTED** [review]
looks good, but please find a japanese reviewer for the textual check

**Comment by rohityadavcloud** [comment]
cc @shwstppr can we get a Japanese translator to glance over this (maybe spend 10-15mins to quickly say LGTM proxied through you ;) )

**Comment by rohityadavcloud** [comment]
I've used the translated changes to translated via Google Translate to check, it LGTM but let's see if @shwstppr can get a LGTM as well
Screenshot 2022-04-05 at 1 23 58 PM

**Review by rohityadavcloud — APPROVED** [review]

**Comment by shwstppr** [comment]
@rohityadavcloud will do
@tajisoft will you add more changes? I see some strings are yet to be translated
Screenshot from 2022-04-05 13-44-43

**Comment by tajisoft** [comment]
@rohityadavcloud @shwstppr Thank you for comment. I'm thinking of checking all the translations. Currently it is finished until 1/4. It will take a little longer, but I will do it all. When all is done and the review is ready, change to Wait for review status.

**Comment by rohityadavcloud** [comment]
Thanks for updating @tajisoft pl ping us when your PR is ready for review.

**Comment by tajisoft** [comment]
@rohityadavcloud I got it.

**Comment by tajisoft** [comment]
@rohityadavcloud @shwstppr Translation is complete. I think this makes it easier for Japanese users. Unfortunately I couldn't find another Japanese developer so I couldn't double check. I will continue to fix any translations that are strange in context.

**Comment by rohityadavcloud** [comment]
Thanks @tajisoft I'll see if we've somebody to review your changes, @shwstppr can we request your translator friend to spare 5-10mins to skim/check this PR 😉?

**Review by DaanHoogland — APPROVED** [review]

File: ui/public/locales/ja_JP.json [hunk 1]
```diff
@@ -1,2367 +1,3534 @@
...
+"error.please.specify.physical.network.tags": "この物理ネットワークのタグを指定しなければ、ネットワークオファリングは使用できません。",
+"error.release.dedicate.cluster": "専用クラスター解除に失敗しました。",
+"error.release.dedicate.host": "専用ホスト解除に失敗しました。",
+"error.release.dedicate.pod": "専用ポッド解除に失敗しました。",
+"error.release.dedicate.zone": "専用ゾーン解除に失敗しました。",
+"error.session.expired": "セッションの有効期限が切れました。",
+"error.unable.to.proceed": "継続できません。管理者に連絡してください。",
+"error.unable.to.reach.management.server": "管理サーバーと通信できません",
+"error.unresolved.internet.name": "インターネット名を解決できません。",
+"firewall.close": "Firewall",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:30]:
```suggestion
"firewall.close": "ファイアウォール閉め",
```

File: ui/public/locales/ja_JP.json [hunk 2]
```diff
@@ -1,2367 +1,3534 @@
...
+"hint.type.part.storage.tag": "ストレージタグの種類",
+"icmp.code.desc": "すべてのICMPコードを許可する場合-1を指定してください。",
+"icmp.type.desc": "すべてのICMPタイプを許可する場合-1を指定してください",
+"image.directory": "画像ディレクトリ",
+"inline": "直列",
+"label.about": "バージョン情報",
+"label.about.app": "CloudStackについて",
+"label.accept": "Accept",
+"label.accept.project.invitation": "プロジェクトへの招待の承諾",
+"label.access": "Access",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:48]:
```suggestion
"label.access": "アクセス",
```

File: ui/public/locales/ja_JP.json [hunk 3]
```diff
@@ -1,2367 +1,3534 @@
...
+"hint.no.storage.tags": "ストレージタグが見つかりません",
+"hint.type.part.host.tag": "ホストタグの種類",
+"hint.type.part.storage.tag": "ストレージタグの種類",
+"icmp.code.desc": "すべてのICMPコードを許可する場合-1を指定してください。",
+"icmp.type.desc": "すべてのICMPタイプを許可する場合-1を指定してください",
+"image.directory": "画像ディレクトリ",
+"inline": "直列",
+"label.about": "バージョン情報",
+"label.about.app": "CloudStackについて",
+"label.accept": "Accept",
```
[the remark this answers is not in the document]
Comment (tajisoft) [reply ui/public/locales/ja_JP.json]: @alkasingh03 Thank you for review!!
"reject" -> "拒否"
"accept" -> "承諾"
I think this pair would be good.

File: ui/public/locales/ja_JP.json [hunk 4]
```diff
@@ -1,2367 +1,3534 @@
...
-"label.baremetalcpucores": "CPU \u30b3\u30a2\u6570",
-"label.baremetalmac": "\u30db\u30b9\u30c8\u306e MAC",
-"label.baremetalmemory": "\u30e1\u30e2\u30ea (MB)",
-"label.basic": "\u57fa\u672c",
-"label.basic.mode": "\u57fa\u672c\u30e2\u30fc\u30c9",
+"label.apply": "適用",
+"label.archive": "アーカイブ",
+"label.archive.alerts": "アラートのアーカイブ",
+"label.archive.events": "イベントのアーカイブ",
+"label.as.default": "as default",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:461]: デフォルトとして

File: ui/public/locales/ja_JP.json [hunk 5]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.assign": "割り当て",
+"label.assign.instance.another": "ほかのアカウントへのインスタンスの割り当て",
+"label.assign.to.load.balancer": "ロード バランサーにインスタンスを割り当てています",
+"label.assign.vms": "仮想マシンの割り当て",
+"label.assigning.vms": "Assigning VMs",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:466]:
```suggestion
"label.assigning.vms": "VMを割り当て中",
```

File: ui/public/locales/ja_JP.json [hunk 6]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.by.zone": "ゾーン",
+"label.bypassvlanoverlapcheck": "VLAN id/rangeのオーバーラップをバイパス",
+"label.cachemode": "書き込みキャッシュの種類",
+"label.cancel": "キャンセル",
+"label.capacity": "処理能力",
+"label.capacity.iops": "処理能力(IOPS)",
+"label.capacitybytes": "処理能力(バイト)",
+"label.capacityiops": "IOPS合計",
+"label.category": "カテゴリー",
+"label.certchain": "Chain",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:550]:
```suggestion
"label.certchain": "チェイン",
```

File: ui/public/locales/ja_JP.json [hunk 7]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.esppolicy": "ESPポリシー",
+"label.esx.host": "ESX/ESXiホスト",
+"label.event": "イベント",
+"label.event.archived": "イベントがアーカイブされました",
+"label.event.deleted": "イベントが削除されました",
+"label.event.timeline": "イベントタイムライン",
+"label.events": "イベント",
+"label.every": "毎",
+"label.example": "例",
+"label.example.plugin": "ExamplePlugin",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:947]:
```suggestion
"label.example.plugin": "例プラッグイン",
```

File: ui/public/locales/ja_JP.json [hunk 8]
```diff
@@ -1,2367 +1,3534 @@
...
-"label.format": "\u5f62\u5f0f",
-"label.friday": "\u91d1\u66dc\u65e5",
-"label.full": "\u5b8c\u5168",
-"label.full.path": "\u30d5\u30eb \u30d1\u30b9",
+"label.f5.details": "F5の詳細",
+"label.f5.ip.loadbalancer": "F5 Big Ipロードバランサー",
+"label.failed": "失敗",
+"label.featured": "おすすめ",
+"label.fetch.latest": "最新情報の取得",
+"label.files": "Alternate Files to Retrieve",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:962]:
```suggestion
"label.files": "取得する代替ファイル",
```

File: ui/public/locales/ja_JP.json [hunk 9]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.filterby": "フィルター",
+"label.fingerprint": "フィンガープリント",
+"label.firewall": "ファイアウォール",
+"label.firewallrule": "ファイアウォールルール",
+"label.firstname": "名",
+"label.firstname.lower": "名",
+"label.fix.errors": "エラーを修正",
+"label.fixed": "修正されたオファリング",
+"label.for": "for",
+"label.forbidden": "Forbidden",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:975]:
```suggestion
"label.forbidden": "禁断",
```

File: ui/public/locales/ja_JP.json [hunk 10]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.forced": "強制",
+"label.forceencap": "ESPパケットのUDPカプセル化を強制する",
+"label.forgedtransmits": "偽造送信",
+"label.format": "形式",
+"label.free": "Free",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:980]:
```suggestion
"label.free": "自由",
```

File: ui/public/locales/ja_JP.json [hunk 11]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.installwizard.addzoneintro.subtitle": "ゾーンについて",
+"label.installwizard.addzoneintro.title": "ゾーンを追加しましょう",
+"label.installwizard.click.launch": "[起動] をクリックしてください。",
+"label.installwizard.subtitle": "このガイドツアーはCloudStack&#8482;環境のセットアップに役立ちます",
+"label.installwizard.title": "CloudStack&#8482;へようこそ",
+"label.instance": "インスタンス",
+"label.instance.groups": "インスタンスグループ",
+"label.instance.name": "インスタンス名",
+"label.instance.scaled.up": "インスタンスを選択したオファリングにスケールする",
+"label.instancename": "内部名",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1140]:
```suggestion
"label.instancename": "インスタンス名",
```

File: ui/public/locales/ja_JP.json [hunk 12]
```diff
@@ -1,2367 +1,3534 @@
...
-"label.metrics.cpu.allocated": "CPU Allocation",
-"label.metrics.cpu.usage": "CPU Usage",
+"label.lun.number": "LUN番号",
+"label.lxcnetworklabel": "LXCのトラフィックラベル",
+"label.macaddress": "MACアドレス",
+"label.macaddress.example": "MACアドレス。例: 01:23:45:67:89:ab",
+"label.macaddresschanges": "MACアドレス変更",
+"label.maclearning": "MAC学習",
+"label.macos": "MacOS",
+"label.make": "Make",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1350]:
```suggestion
"label.make": "作成",
```

File: ui/public/locales/ja_JP.json [hunk 13]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.ostypeid": "OSの種類",
+"label.ostypename": "OSの種類",
+"label.other": "その他",
+"label.outofbandmanagement": "Out-of-band管理",
+"label.outofbandmanagement.action.issue": "Out-of-band管理電源アクションを発行",
+"label.outofbandmanagement.changepassword": "Change Out-of-band管理パスワードを変更 Password",
+"label.outofbandmanagement.configure": "Out-of-band管理設定",
+"label.outofbandmanagement.disable": "Out-of-band管理無効化",
+"label.outofbandmanagement.enable": "Out-of-band管理有効化",
+"label.overprovisionfactor": "Overprovisioning Factor",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1618]:
```suggestion
"label.overprovisionfactor": "オーバープロビジョニング要因",
```

File: ui/public/locales/ja_JP.json [hunk 14]
```diff
@@ -1,2367 +1,3534 @@
...
-"label.removing": "\u524a\u9664\u3057\u3066\u3044\u307e\u3059",
-"label.removing.user": "\u30e6\u30fc\u30b6\u30fc\u3092\u524a\u9664\u3057\u3066\u3044\u307e\u3059",
-"label.replace.acl": "ACL \u306e\u7f6e\u304d\u63db\u3048",
-"label.replace.acl.list": "ACL \u4e00\u89a7\u306e\u7f6e\u304d\u63db\u3048",
-"label.required": "\u5fc5\u9808\u3067\u3059",
+"label.rbdid": "Cephxユーザー",
+"label.rbdmonitor": "Cephモニター",
+"label.rbdpool": "Cephプール",
+"label.rbdsecret": "Cephxシークレット",
+"label.read": "Read",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1814]:
```suggestion
"label.read": "読み取り",
```

File: ui/public/locales/ja_JP.json (hunk 4 above)
Comment (DaanHoogland) [reply ui/public/locales/ja_JP.json:461]: @alkasingh03 , do you mean that 
```suggestion
"label.as.default": "デフォルトとして",
```

File: ui/public/locales/ja_JP.json [hunk 15]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.read.io": "Read (IO)",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1815]:
```suggestion
"label.read.io": "読み取り (IO)",
```

File: ui/public/locales/ja_JP.json [hunk 16]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.readonly": "Read-Only",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:1816]:
```suggestion
"label.readonly": "読み取り専用",
```

File: ui/public/locales/ja_JP.json [hunk 17]
```diff
@@ -1,2367 +1,3534 @@
...
-"label.sourcecidr": "\u9001\u4fe1\u5143 CIDR",
-"label.sourceipaddress": "\u9001\u4fe1\u5143 IP \u30a2\u30c9\u30ec\u30b9",
-"label.sourcenat": "\u9001\u4fe1\u5143 NAT",
-"label.sourcenattype": "\u30b5\u30dd\u30fc\u30c8\u3055\u308c\u308b\u9001\u4fe1\u5143 NAT \u306e\u7a2e\u985e",
-"label.sourceport": "\u9001\u4fe1\u5143\u30dd\u30fc\u30c8",
-"label.specify.vxlan": "VXLAN \u3092\u6307\u5b9a\u3059\u308b",
-"label.specifyipranges": "IP \u30a2\u30c9\u30ec\u30b9\u306e\u7bc4\u56f2\u306e\u6307\u5b9a",
-"label.specifyvlan": "VLAN \u3092\u6307\u5b9a\u3059\u308b",
-"label.sr.name": "SR \u540d\u30e9\u30d9\u30eb",
+"label.sharewith": "Share With",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:2049]:
```suggestion
"label.sharewith": "共有",
```

File: ui/public/locales/ja_JP.json [hunk 18]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.show.ingress.rule": "受信ルールの表示",
+"label.showing": "Showing",
```
Comment (alkasingh03) [inline ui/public/locales/ja_JP.json:2051]:
```suggestion
"label.showing": "表示中",
```

**Review by alkasingh03 — COMMENTED** [review]
＠tajisoft some more suggestions added.
Good work translating huge list! よく頑張りましたね。

**Comment by alkasingh03** [comment]
@rohityadavcloud Rohit-san reviewed changes, added suggestions for some missing strings :)

**Comment by tajisoft** [comment]
> ＠tajisoft some more suggestions added. Good work translating huge list! よく頑張りましたね。

Thank you for review and a lot of suggestions!! I'll fix it soon!

File: ui/public/locales/ja_JP.json (hunk 9 above)
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:975]: The word "forbidden" is so well known that I decided not to translate it.

File: ui/public/locales/ja_JP.json (hunk 11 above)
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:1140]: @alkasingh03 
English is using "Internal name" here. I think [Internal = "内部" ] is good. What do you think?
「"label.instancename": "Internal name",」

File: ui/public/locales/ja_JP.json [hunk 19]
```diff
@@ -1,2367 +1,3534 @@
...
+"label.alerts": "アラート",
+"label.algorithm": "アルゴリズム",
+"label.all": "すべて",
+"label.all.zone": "すべてのゾーン",
+"label.allocated": "割り当て済み",
+"label.allocatediops": "IOPS割り当て済",
+"label.allocationstate": "割り当て状態",
+"label.allow": "許可",
+"label.allowuserdrivenbackups": "ユーザーバックアップを許可",
+"label.annotated.by": "Annotator",
```
[the remark this answers is not in the document]
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:446]: I use "アノテーター" here.

File: ui/public/locales/ja_JP.json (hunk 7 above)
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:947]: I use "プラグイン例" here.

File: ui/public/locales/ja_JP.json (hunk 11 above)
Comment (tajisoft) [reply ui/public/locales/ja_JP.json:1140]: I keep to use "内部名" here.

**Comment by tajisoft** [comment]
Thank you to everyone who cooperated! I'm very happy that the translation has been completed and it has become more friendly to Japanese users!

**Comment by tajisoft** [comment]
I think I made a mistake in git operation. Can I open a new PR?

**Comment by tajisoft** [comment]
I opened a new PR #6332.

**Comment by DaanHoogland** [comment]
> I opened a new PR #6332.

thanks @tajisoft , most of the time git mistakes are repairable. Iĺl close this PR as you already opened a new one.

**Comment by tajisoft** [comment]
> > I opened a new PR #6332.
> 
> thanks @tajisoft , most of the time git mistakes are repairable. Iĺl close this PR as you already opened a new one.

@DaanHoogland  Thank you for comment! Next up will improve my Git skills!

**Pull request closed without merging**